src/allmydata/web/root.py

```diff
@@ -1,7 +1,8 @@
 import time, os
+import simplejson
```

Comment:
yeah, we should definitely switch to stdlib `json`. I'm not sure whether it'd be better for new code to keep using `simplejson`, and then switch everything in one fell swoop later. Or to have new code use `json` and do the cutover piecemeal.